src/@next/AuthenticationInput/AuthenticationInputStyle.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import { space4 } from '../utilities/spacing';
```

Comment:
@marthayaputra As the component in Figma is incorrectly wrapped twice, the 4px you saw was incorrect. If you click into the second layer, you'll see 8px, so please help to change `space4` to `space8`.

Reply:
Fixed